proto/minder/v1/minder.proto

```diff
@@ -929,7 +929,7 @@ service InviteService {
         };
 
         option (rpc_options) = {
-            target_resource: TARGET_RESOURCE_PROJECT
+            target_resource: TARGET_RESOURCE_NONE
```

Comment:
This is needed so we can make the method explicitly available to unauthenticated users.